packages/@aws-cdk/aws-certificatemanager/README.md

```diff
@@ -39,25 +35,49 @@ in the AWS Certificate Manager User Guide.
 
 ### DNS validation
 
-DNS-validated certificates are validated by configuring appropriate DNS
-records for your domain.
+If Amazon Route 53 is your DNS provider for the requested domain, the DNS record can be
+created automatically:
+
+```ts
+new Certificate(this, 'Certificate', {
+  domainName: 'hello.example.com',
+  validation: CertificateValidation.fromDns(myHostedZone), // Route 53 hosted zone
+});
+```
 
-See [Validate with DNS](https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-validate-dns.html)
+Otherwise DNS records must be added manually and the stack will not complete
+creating until the records are added.
+
+```ts
+new Certificate(this, 'Certificate', {
+  domainName: 'hello.example.com',
+  validation: CertificateValidation.fromDns(), // Records must be added manually
+});
+```
+
+See also [Validate with DNS](https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-validate-dns.html)
 in the AWS Certificate Manager User Guide.
 
-### Automatic DNS-validated certificates using Route53
+When working with multiple domains, use the `CertificateValidation.fromDnsMultiZone()`:
+
+[multiple domains DNS validation](test/example.dns.lit.ts)
+
+Use the `DnsValidatedCertificate` construct for cross-region certificate creation:
+
+```ts
+new DnsValidatedCertificate(this, 'CrossRegionCertificate', {
```

Comment:
Should we do:

```ts
export class CrossRegionDnsValidatedCertificate extends DnsValidatedCertificate {
  constructor(scope: cdk.Construct, id: string, props: DnsValidatedCertificateProps) {
    super(scope, id, props);
  }
}
```

to get a better naming?

Reply:
I don't think it's worth it, especially considering cross-region support is on the roadmap for CloudFormation. Let's just leave it as-is and deprecate it once we can.